impeller/scene/node.cc

```diff
@@ -190,7 +194,7 @@ void Node::UnpackFromFlatbuffer(
   }
 }
 
-Node::Node() = default;
+Node::Node() : name_(SPrintF("__node%" PRIu64, kNextNodeID.fetch_add(1))){};
```

Comment:
You could just use [`operator++`](https://en.cppreference.com/w/cpp/atomic/atomic/operator_arith) for these atomics which does the same thing and perhaps is a bit clearer.

Reply:
Ooh, that's nice. Done.